ddev/src/ddev/cli/meta/scripts/generate_metrics.py

```diff
@@ -13,14 +13,24 @@
 
 @click.command('generate-metrics', short_help='Generate metrics with fake values for an integration')
 @click.argument('integration')
-@click.option('--api-url', default="api.datadoghq.com", help='The API URL to use, e.g. "api.datadoghq.com"')
-@click.option('--api-key', required=True, help='Your API key')
+@click.option(
+    '--site',
+    help='The datadog SITE to use, e.g. "datadoghq.com". If not provided we will use ddev config org settings.',
+)
+@click.option('--api-key', help='The API key. If not provided we will use ddev config org settings.')
 @click.pass_obj
-def generate_metrics(app: Application, integration: str, api_url: str, api_key: str):
+def generate_metrics(app: Application, integration: str, site: str, api_key: str):
     """Generate metrics with fake values for an integration
 
+    You can provide the site and API key as options:
+
+    \b
+    `$ ddev meta scripts generate-metrics ray --site <URL> --api-key <API_KEY>`
+
+    It's easier however to switch ddev's org setting temporarily:
+
     \b
-    `$ ddev meta scripts generate-metrics ray --api-url <URL> --api-key <API_KEY>`
+    `$ ddev -o <your-org> meta scripts generate-metrics ray`
```

Comment:
Is it org id? or org name? Also, should we replace ray by  <integration> for clarity?

Reply:
It's what you see when you do `ddev config show` and look for tables `[orgs.your-org]`.

Reply:
I agree there's no clear terminology for this, partly cuz the whole org thing isn't documented well in general.